xx.md

```diff
@@ -0,0 +1,57 @@
+NIP-XX
+=======
+
+Invite Links
+------------
+
+This NIP defines a new bech32-encoded entity, `ninvite1` and a new event representing an invite code which
+will grant access to a given relay.
+
+## Invite entity
+
+A `ninvite` is a bech32-encoded entity which includes one or more pubkeys, relay urls, and group ids.
+
+```typescript
+InvitePointer = {
+  people?: string[]
+  relays?: {
+    url: string
+    claim?: string
+  }[]
+  groups?: {
```

Comment:
Sorry for the basic question, but what exactly does "groups" refer to?

Reply:
Any replaceable event that can be "joined". So my intention here is NIP 72 communities and (draft) NIP 87 groups, but anything else analogous that comes up.

Reply:
Is the reference to those events set to "address"?

Reply:
Yep, same value as an `a` tag

Reply:
I understand and thanks for your answers.